.github/components-path-filters.yml

```diff
@@ -273,13 +273,13 @@ interactive_ai/workflows/geti_domain/train:
   - Makefile.shared
   - .github/workflows/component.yml
 # TODO: Re-enable CVS-166868
-#interactive_ai/workflows/otx_domain/trainer/otx_v2:
-#  - interactive_ai/workflows/otx_domain/trainer/otx_v2/**
-#  - libs/telemetry_tools/**
-#  - libs/kafka_tools/**
-#  - Makefile.shared-python
-#  - Makefile.shared
-#  - .github/workflows/component.yml
+interactive_ai/workflows/otx_domain/trainer/otx_v2:
+  - interactive_ai/workflows/otx_domain/trainer/otx_v2/**
+  - libs/telemetry_tools/**
+  - libs/kafka_tools/**
+  - Makefile.shared-python
+  - Makefile.shared
+  - .github/workflows/component.yml
```

Comment:
Note (as a reminder to myself): I temporarily enabled this for testing purposes. Should be re-disabled before merging.